inc/functions/cloudflare.php

```diff
@@ -737,6 +737,22 @@ function rocket_auto_purge_cloudflare_by_url( $post, $purge_urls, $lang ) {
 		return;
 	}
 
+	// Add home URL and feeds URLs to Cloudflare clean cache URLs list.
+	$home_url     = get_rocket_i18n_home_url( $lang );
+	$purge_urls[] = $home_url;
+	$feed_urls    = [];
+	$feed_urls[]  = get_feed_link();
+	$feed_urls[]  = get_feed_link( 'comments_' );
+
+	/**
+	 * Filter the home feeds urls
+	 *
+	 * @since 2.7
+	 * @param array     $urls The urls of the home feeds.
+	*/
```

Comment:
since the filter already exists somewhere else, you can reference it that way instead:
`// this filter is documented in inc/functions/files.php`

Reply:
Applied the same change in v3.5 refactored code.